src-cpp/server42/Dockerfile.protobuf

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:18.04
```

Comment:
can we have 20.04? Recently we have an issue with "missing" ubuntu repositories for outdated ubuntu release.

Reply:
+1

Reply:
+1

Reply:
agree

Reply:
agree

Reply:
done